packages/netlify-cms-backend-bitbucket/src/API.js

```diff
@@ -159,16 +159,12 @@ export default class API {
       unsentRequest.withMethod('POST'),
       unsentRequest.withBody(formData),
       this.request,
-      then(() => ({ ...item, uploaded: true })),
+      then(() => ({ ...item })),
```

Comment:
The `uploaded ` flag was used internally to let the asset proxy know if to use the public folder when loading the image